views/friendlycode/templates/nav-options.html

```diff
@@ -1,7 +1,7 @@
 <div class="editor-nav">
 
   <!-- FILE TREE -->
-  <div class="filetree-pane-nav">
+  <div class="filetree-pane-nav" style="display:none;" class="hide">
```

Comment:
Not necessary, but added for consistency.
This element needs an inline display:none in order to hide it on load (the class adds the CSS too late, and creates a bad load visual).
 Everything else uses class="hide" except this element.